.github/workflows/go.yml

```
@@ -88,7 +88,7 @@ jobs:
       # version, but we prefer this action because it leaves 'annotations' (i.e.
       # it comments on PRs to point out linter violations).
       - name: Lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
           version: ${{ env.GOLANGCI_VERSION }}
```

```suggestion
          version: ${{ env.GOLANGCI_VERSION }}
          skip-go-installation: true
```

Let's try add this as well